lib/accumulate_distribute/meta/init_state.js

```diff
@@ -22,9 +23,9 @@ const initState = (args = {}) => {
   const orderAmounts = genOrderAmounts.gen(args)
 
   let totalAmount = 0
-  orderAmounts.forEach(a => { totalAmount += a })
+  orderAmounts.forEach(a => { totalAmount = nBN(totalAmount).plus(a).toNumber() })
 
-  if (Math.abs(totalAmount - amount) > DUST) {
+  if (Math.abs(nBN(totalAmount).minus(amount).toNumber()) > DUST) {
```

Comment:
total amount is just used internally, for a check on dust, no need for that. 

(you can revert this file back)